### DISCUSSION TITLE
Save content and Render

### DISCUSSION THREAD

**Discussion by devopg** [Milkdown/milkdown#683] [Q&A]
1) How can I take all the content from editor to save on server by call my function?

```
const { editor } = useEditor(root =>
	Editor.make()
		...
); 
function DoAction(){
     let content = editor.takeAllContent(); // ???
}
```
2) How can I render saved content to clear div (not into editor) loaded from server??

> It is strange that the simplest things are not described in the documentation

**Comment by Zenyet** [comment]
https://milkdown.dev/interacting-with-editor#markdown-listener. I think the answer is here..
